docs/plugins/outputs/elastic_app_search.asciidoc

```diff
@@ -1,14 +1,14 @@
 :plugin: elastic_app_search
 :type: output
 :default_plugin: 1
-:no_codec:
+:default_codec: plain
 
 ///////////////////////////////////////////
 START - GENERATED VARIABLES, DO NOT EDIT!
 ///////////////////////////////////////////
-:version: v1.0.0
-:release_date: 2019-06-12
-:changelog_url: https://github.com/logstash-plugins/logstash-output-elastic_app_search/blob/v1.0.0/CHANGELOG.md
+:version: v1.0.0.beta1
+:release_date: 2018-11-06
+:changelog_url: https://github.com/logstash-plugins/logstash-output-elastic_app_search/blob/v1.0.0.beta1/CHANGELOG.md
 :include_path: ../../../../logstash/docs/include
```

Comment:
Regression from lockfile. Codec setting. I should probably fix this in generated output?